### ISSUE TITLE
The library can throw Newtonsoft.Json.JsonReaderException in some cases when WP_DEBUG_DISPLAY is true in wordpress configuration

### ISSUE THREAD

**Issue by navjot50** [wp-net/WordPressPCL#273]
I tried to upload an image using WordpressPCL and I got a Newtonsoft.Json.JsonReaderException with a message "Unexpected character encountered while parsing value: <. Path '', line 0, position 0.". I found out that the image was successfully uploaded on the wordpress server but not assigned to `MediaItem item = await client.Media.Create(imageStream, "img.jpg");`, instead there was the above exception.

Further investigating, I found out that the json response from the wordpress server was preceded by PHP warnings markup. A cut down version of the response looked like:

```
<br />
<font size='1'><table class='xdebug-error xe-warning' dir='ltr' border='1' cellspacing='0' cellpadding='1'>
<tr><th align='left' bgcolor='#f57900' colspan="5"><span style='background-color: #cc0000; color: #fce94f; font-size: x-large;'>( ! )</span> Warning: exif_read_data(img.jpg): Incorrect APP1 Exif Identifier Code in C:\Users\Scintec AG\Sites\www.scintec.dev.cc\wp-admin\includes\image.php on line <i>801</i></th></tr>
<tr><th align='left' bgcolor='#e9b96e' colspan='5'>Call Stack</th>
//followed by some more markup here
{"id":2171,"date":"2021-10-06T13:59:55","date_gmt":"2021-10-06T13:59:55" ...} //json to be serialized by library
```

So, basically the library was expecting no markup in front of json. I found out that this markup comes up only when `WP_DEBUG_DISPLAY` is set to true. In development phase this constant is set to true most of the times. So, the library should handle this because the image is uploaded on the server but the json for it could not be deserialized by the library.

I can give a PR to fix this issue.

**Comment by ThomasPe** [comment]
sounds like a plan to me. are there instances where an array is returned from the API?
